README.md

```diff
@@ -2,33 +2,76 @@
  <br/>
  <img src="https://raw.githubusercontent.com/motorlatitude/Drop/master/src/assets/img/Drop%20Logo.png" alt="Drop Logo" width="50%">
  <br/><br/>
- <img src="https://david-dm.org/motorlatitude/drop.svg" alt="Dependency Version">
- <img src="https://img.shields.io/github/package-json/v/motorlatitude/drop" alt="Drop Version">
 </p>
 
+# Drop - Color Picker Tool
 
+<p align="left">
+    <img src="https://travis-ci.com/motorlatitude/Drop.svg?branch=master" alt="Build Status">
+    <img src="https://david-dm.org/motorlatitude/drop.svg" alt="Dependency Version">
+    <img src="https://img.shields.io/github/downloads/motorlatitude/Drop/total?color=green" alt="GitHub All Releases">
+    <img src="https://img.shields.io/github/package-json/v/motorlatitude/drop" alt="Drop Version">
+    <br/>
+</p>
+
+Drop is a color picker made for windows using electron. It's designed for developers and designers as a way to gather,
+tweak and share colors, featuring a rich UI to support it along the way.
 
+- Multi-level Magnification
+- Customizable Shortcuts
+- Different color formats that it can output (CSS RGB, RGBA, HSL, HSLA and Hex with more coming soon!!!)
+- Create and share palettes to keep your different projects organized
+- Automatically puts the clicked colour into your clipboard
+- With more features on the way...
 
 🏗️ **Development Stage**: Alpha
 
+This project is still in its development stage so some aspects of it will not work as expected and will still contain
+bugs.
+
+<p align="center">
+    <img src="https://raw.githubusercontent.com/motorlatitude/Drop/alpha.5/src/assets/img/thumbnail.png" alt="Drop Preview - Showing the history window" width="75%">
+</p>
+
+## Installation
 
-Drop is a color-picker designed for windows using electron. It's designed as a way to gather, tweak and share colors than
-existing tools, featuring a rich UI to support it along the way.
+Installation is simple, just run the setup executable from the latest release.
 
-### ✨ Features
+[🔽 Download Latest Release](https://github.com/motorlatitude/Drop/releases)
 
- - Multi-level Magnification
- - Customizable Shortcuts
- - Different color formats that it can output (CSS RGB, RGBA, HSL, HSLA and Hex with more coming soon!!!)
- - Create palettes to keep your different projects organized
- - Automatically puts the clicked colour into your clipboard
- - With more features on the way...
+Once installed, drop will sit in your taskbar and wait for you to need it. It defaults to starting on system launch
+so it will always be ready to use.
 
-### 🐾 Getting Started
```

Comment:
Should probably have some sort of getting started/guide section to introduce users to features that are maybe not completely obvious.